app/helpers/view_components/widgets/exercise.rb

```diff
@@ -0,0 +1,76 @@
+module ViewComponents
```

Comment:
We might choose to convert this to a React component as we will almost certainly want to use these within other React components.